src/net/server/connection.rs

```diff
@@ -75,6 +75,14 @@ const RESPONSE_WRITE_TIMEOUT: DefMinMax<Duration> = DefMinMax::new(
     Duration::from_secs(60 * 60),
 );
 
+/// Limit on the number of attempts that will be made to overcome
+/// non-fatal write errors.
+///
+/// The value has to be between 0 and 255 with a default of 2. A value of
+/// zero means that one try will be attempted.
+const RESPONSE_WRITE_RETRIES: DefMinMax<u8> =
```

Comment:
I don't understand this. What kind of error do we expect that should be retried. Is there something that is broken in Tokio?

Reply:
I have removed the response write retries config setting. The PR still has a useful additional test that I think is worth keeping.